### DISCUSSION TITLE
版面识别

### DISCUSSION THREAD

**Discussion by whwususu** [PaddlePaddle/PaddleOCR#14070] [Q&A]
请问下，版面分析模型的训练方法我看了半天不得其解，都是提供了各种数据集进行训练，而不是自定义训练集。那么有不有例如通过打标的方式，生成自己的训练集，然后进行训练的。
这个是我看的教程。https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/ppstructure/layout/README_ch.md
谢谢！

**Comment by Liyulingyue** [comment]
需要自行合成，参考paddlex https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/pipeline_usage/tutorials/ocr_pipelines/layout_parsing.md 或者 https://github.com/opendatalab/DocLayout-YOLO
